dev/vagrant/conf/nomad/00-enable_vault.hcl

```diff
@@ -0,0 +1,5 @@
+vault {
+  enabled = true
+  token   = "master"
+  address = "http://127.0.0.1:8200"
```

Comment:
```suggestion
```
You do not need this if they VAULT_TOKEN and VAULT_ADDR is set, do you? I guess it's a lot better for readability though

Reply:
I think we need this since it's making Vault available with Nomad. Don't think they are related to the envs in that regard you describe. Or am I wrong? 🤔

Reply:
@claesgill you are right. 
I am investigating is it a default setting for the Vault or not.

Related issue https://github.com/fredrikhgrelland/vagrant-hashistack/pull/383

Reply:
I could've sworn nomad uses `VAULT_TOKEN` and `VAULT_ADDR` if they are set. I might very well be wrong

Reply:
issue created https://github.com/fredrikhgrelland/vagrant-hashistack/issues/446 . It is a bit different context, but related to our discussion